include/af/defines.h

```diff
@@ -496,6 +503,9 @@ namespace af
 #if AF_API_VERSION >= 34
     typedef af_random_engine_type randomEngineType;
 #endif
+#if AF_API_VERSION >= 35
+    typedef af_canny_threshold CannyThreshold;
```

Comment:
@9prady9 We have been using enums in snakeCase forever. You can't change the style on a whim.